specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2021-05-01-preview/extensionTypes.json

```diff
@@ -0,0 +1,422 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-05-01-preview",
+    "title": "ExtensionTypes",
+    "description": "Use these APIs to create extension resources through ARM, for Kubernetes Clusters."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
```

Comment:
Linting validation is failing with this error: 

```
ERROR (OperationsAPIImplementation/R3023/ARMViolation): Operations API must be implemented for '/providers/Microsoft.KubernetesConfiguration/operations'.
    - file:///D:/GitHub/Azure/azure-rest-api-specs/specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2021-05-01-preview/extensionTypes.json:36:2 ($.paths)
```

@NarayanThiru already has `Microsoft.KubernetesConfiguration/operations` in his changes for this same API preview version [here](https://github.com/NarayanThiru/azure-rest-api-specs/blob/a707a972c537d5c1e25192f2425998ce92495538/specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2021-05-01-preview/extensions.json#L385). We are going to add the new operations I am defining in Narayan's PR